dashing/distribution.yaml

```diff
@@ -2388,6 +2388,26 @@ repositories:
       url: https://github.com/ros2/rcutils.git
       version: dashing
     status: developed
+  realsense2_camera:
+    doc:
+      type: git
+      url: https://github.com/IntelRealSense/realsense-ros.git
+      version: ros2
+    release:
+      packages:
+      - realsense2_camera
+      - realsense2_camera_msgs
+      - realsense2_description
+      tags:
+        release: release/dashing/{package}/{version}
+      url: https://github.com/IntelRealSense/realsense-ros-release.git
+      version: 3.2.1-1
+    source:
+      test_pull_requests: true
```

Comment:
did you follow the steps [here](http://wiki.ros.org/buildfarm/Pull%20request%20testing#Link_Jenkins_and_your_GitHub_repository)?

Reply:
This repo already has the `test_pull_requests` flag on in other distros, so this seems to be ok